inst/pages/19_visualization_techniques.qmd

```diff
@@ -467,10 +456,10 @@ sample_data <- data.frame(clusters = sample_clusters)
 sample_data <- sample_data[samples_ordered, , drop = FALSE] 
 
 # Order data based on 
-tse_phylum_subset <- tse_phylum_subset[ , rownames(sample_data)]
+tse_phylum <- tse_phylum[ , rownames(sample_data)]
 
 # Add sample type data
-sample_data$sample_types <- unfactor(colData(tse_phylum_subset)$SampleType)
+sample_data$sample_types <- unfactor(colData(tse_phylum)$SampleType)
```

Comment:
could be just tse_phylum$SampleType (or altExp(tse, "Phylum", withColData=TRUE)$SampleType